docs/overview/excel.md

```diff
@@ -68,7 +68,15 @@ More details about sharing and unsharing scripts can be in the article [Sharing
 > [!NOTE]
 > Learn more about how scripts are stored in your OneDrive in [Office Scripts file storage and ownership](script-storage.md).
 
-## Connecting Office Scripts to Power Automate
+### Create script buttons (preview)
+
+[!INCLUDE [Public preview](../includes/preview-note.md)]
+
+Help your colleagues find and run your scripts by adding script buttons to a workbook. With any script, go to the **More options (…)** menu and select **Add button**. This creates a button in the workbook that runs the associated script when pressed. It also shares the script with the workbook, so everyone with access can use your helpful automation.
```

Comment:
Users can add buttons through two ways (1) the More options menu in the Script Details pane and (2) the More options menu in the Code Editor. Suggest including that info somewhere in here e.g. "With any script, go to the "More options" menu in either your Code Editor or Script Details pane and select "Add button".

Reply:
Nit: should we add that it shares the script with everyone with write/edit access? a user can't run a button in read-only mode - this behavior is consistent with running scripts though